crates/holochain/src/conductor/conductor.rs

```diff
@@ -1508,13 +1509,17 @@ mod app_impls {
 mod cell_impls {
     use super::*;
     impl Conductor {
-        pub(crate) async fn cell_by_id(&self, cell_id: &CellId) -> ConductorResult<Arc<Cell>> {
+        pub(crate) async fn cell_by_id(&self, cell_id: &CellId, require_network_ready: bool) -> ConductorResult<Arc<Cell>> {
             // Can only get a cell from the running_cells list
             if let Some(cell) = self
                 .running_cells
-                .share_ref(|c| c.get(cell_id).map(|i| i.cell.clone()))
+                .share_ref(|c| c.get(cell_id).cloned())
```

Comment:
This doesn't need to be cloned, I believe?
```suggestion
                .share_ref(|c| c.get(cell_id))
```

Reply:
Yes you're right. I do need to clone the cell status into the error message but not the while struct

Reply:
Actually no, this fails because it complains about lifetimes, I've had to put this back